### ISSUE TITLE
background videos aren't respecting prefers reduced motion

### ISSUE THREAD

**Issue by likello** [osu-wams/madrone#246]
I toggle off "show animations in windows" and the background video still plays and then I am unable to pause the video.

the function isn't valid on page load, not tied to the pause click. what I think is happening is that there is a slight delay in loading the background videos via layout builder. this doesn't happen on the homepage with my custom built hero cover. I think the check is happening before the video element is present.

here is the error: 
Uncaught TypeError: videoList.forEach is not a function
    reducedMotionCheck https://engineering.oregonstate.edu/sites/engineering.oregonstate.edu/files/js/js_0m1A7SXQBO536Q6rIUUzn9BOjE5TqN7GgbhoChvAhy0.js?scope=footer&delta=2&language=en&theme=madrone&include=eJyNVVuS4yAMvJDHrtrL7CclQLaZAKIQJJM9_co4zs7DdvYnwaIFklotgBmLcvEdTaE8GOYBbpAtq9H5grmDnwDtyVyUxSkjqpRpyhBYGQoJTNlzMIRqpoBK11Io8inGQr6oQBb3UIiG1Q21i5Dl30ZkC3dVXPGoCn7sXj8ilJrRqoSUGo53cZMnDV6BMZStO4jzAVrCrcV5V-5KzCdIi1fFrqAaEa0GqdxIdFDZhwvGyUXE7OKkjJChWsVPHEaH3ioPGr3se3E-A7frH1yI64c6ATtitQQ9ZarRKhdgwsXnxCWAiypgrGcY0k6YSGDtkuT5gYyQzaw0nYKWEnPJ1SxcnwELFMfFGVndpVTTCXZj-DCbDFbxTDcDjI2q3U6YnUX1S8pnQYIsLuDK2B424iTxXY-7-R-A0mErC7EzXYVlHEex7kZFIldJbm8rkZWMinR5LBiL8DTh7hkbUChiuXOtwChecEMWNQ-f1r3odvnkbiKahH2I4O9ChNTxm6Ejrspc0Dq5SCVfRQ8SsBidWXR5uL0NmAaoXCis2mm7b6vlbbV023ErFw2wfHbswjIlEqWaNu8dW8d3LhgGLdR3AWymuKbLzy8tSpOWhPS0rH31tnXeE9B6EaWeLZf-k-IMecq9rk7a5Qi-tMr_A8G7KUpHlxfghzhfoALkFvEpSMtAxVfBscnk_daxP8Ae7lSLambMm5OoXt6e7ym1VnjhGWRKzeim-VUdZO6IxsHS7QEM0ngwrO9jbyAVeSu6q8MbD-23l9ereuy0hz_3QTvq4R0-viCa4T_Yb1d9uZYwaLT9SDlAWZ4Rrgnz6Hgenqs92zoPXNPz7j4H8F44QIz7-zUlkim7JHuwj5mrfsrbMD6ELavfMvkk4nz5C_1dXgg:35
    <anonymous> https://engineering.oregonstate.edu/sites/engineering.oregonstate.edu/files/js/js_0m1A7SXQBO536Q6rIUUzn9BOjE5TqN7GgbhoChvAhy0.js?scope=footer&delta=2&language=en&theme=madrone&include=eJyNVVuS4yAMvJDHrtrL7CclQLaZAKIQJJM9_co4zs7DdvYnwaIFklotgBmLcvEdTaE8GOYBbpAtq9H5grmDnwDtyVyUxSkjqpRpyhBYGQoJTNlzMIRqpoBK11Io8inGQr6oQBb3UIiG1Q21i5Dl30ZkC3dVXPGoCn7sXj8ilJrRqoSUGo53cZMnDV6BMZStO4jzAVrCrcV5V-5KzCdIi1fFrqAaEa0GqdxIdFDZhwvGyUXE7OKkjJChWsVPHEaH3ioPGr3se3E-A7frH1yI64c6ATtitQQ9ZarRKhdgwsXnxCWAiypgrGcY0k6YSGDtkuT5gYyQzaw0nYKWEnPJ1SxcnwELFMfFGVndpVTTCXZj-DCbDFbxTDcDjI2q3U6YnUX1S8pnQYIsLuDK2B424iTxXY-7-R-A0mErC7EzXYVlHEex7kZFIldJbm8rkZWMinR5LBiL8DTh7hkbUChiuXOtwChecEMWNQ-f1r3odvnkbiKahH2I4O9ChNTxm6Ejrspc0Dq5SCVfRQ8SsBidWXR5uL0NmAaoXCis2mm7b6vlbbV023ErFw2wfHbswjIlEqWaNu8dW8d3LhgGLdR3AWymuKbLzy8tSpOWhPS0rH31tnXeE9B6EaWeLZf-k-IMecq9rk7a5Qi-tMr_A8G7KUpHlxfghzhfoALkFvEpSMtAxVfBscnk_daxP8Ae7lSLambMm5OoXt6e7ym1VnjhGWRKzeim-VUdZO6IxsHS7QEM0ngwrO9jbyAVeSu6q8MbD-23l9ereuy0hz_3QTvq4R0-viCa4T_Yb1d9uZYwaLT9SDlAWZ4Rrgnz6Hgenqs92zoPXNPz7j4H8F44QIz7-zUlkim7JHuwj5mrfsrbMD6ELavfMvkk4nz5C_1dXgg:35
js_0m1A7SXQBO536Q6rIUUzn9BOjE5TqN7GgbhoChvAhy0.js:35:12407

**Comment by emerham** [comment]
This has already been fixed in #244. Will be in next release.
